core/store/tx_manager.go

```diff
@@ -378,6 +389,24 @@ func (txm *EthTxManager) BumpGasUntilSafe(hash common.Hash) (*models.TxReceipt,
 	return nil, Unconfirmed, merr
 }
 
+func (txm *EthTxManager) checkDBForConfirmation(tx *models.Tx) (*models.TxReceipt, AttemptState, error) {
+	later, err := txm.orm.FindLaterConfirmedTx(tx)
+	if err != nil {
+		return nil, Unknown, errors.Wrap(err, "BumpGasUntilSafe checkDBForConfirmation")
+	} else if later == nil {
+		return nil, Unconfirmed, nil
+	}
+
+	tx.Confirmed = true
+	tx.Hash = utils.EmptyHash
```

Comment:
Do we not care about this hash being correct?

Reply:
If it errors, I don't think so. There's not an easy way for us to find the correct hash.